File: src/core/paths.js
import path from "node:path";
import { fileURLToPath } from "node:url";

const currentFile = fileURLToPath(import.meta.url);
const srcRoot = path.dirname(path.dirname(currentFile));

export const projectRoot = path.dirname(srcRoot);
export const defaultIocDir = path.join(projectRoot, "data", "ioc");
export const defaultRulesFile = path.join(projectRoot, "data", "rules", "yara-lite-rules.json");
export const defaultStateDir = path.join(projectRoot, ".antiai");
export const defaultVirusTotalConfig = path.join(projectRoot, "config", "virustotal.json");

File: pyantiai/paths.py
from pathlib import Path


PROJECT_ROOT = Path(__file__).resolve().parent.parent
DEFAULT_IOC_DIR = PROJECT_ROOT / "data" / "ioc"
DEFAULT_RULES_FILE = PROJECT_ROOT / "data" / "rules" / "yara-lite-rules.json"
DEFAULT_STATE_DIR = PROJECT_ROOT / ".antiai"
DEFAULT_VIRUSTOTAL_CONFIG = PROJECT_ROOT / "config" / "virustotal.json"
